src/DDTrace/Integrations/Guzzle/V5/GuzzleIntegrationLoader.php

```diff
@@ -0,0 +1,77 @@
+<?php
+
+namespace DDTrace\Integrations\Guzzle\V5;
+
+use DDTrace\Contracts\Span;
+use DDTrace\Http\Urls;
+use DDTrace\Integrations\Guzzle\AbstractGuzzleIntegrationLoader;
+use DDTrace\Tag;
+
+/**
+ * Concrete integration loader for Guzzle V5
+ */
+final class GuzzleIntegrationLoader extends AbstractGuzzleIntegrationLoader
+{
+    /**
+     * @return string
+     */
+    protected function getMethodToTrace()
+    {
+        return 'send';
+    }
+
+    /**
+     * @param Span $span
+     * @param \GuzzleHttp\Message\RequestInterface $request
+     */
+    protected function setUrlTag(Span $span, $request)
+    {
+        if (!is_a($request, '\GuzzleHttp\Message\RequestInterface')) {
```

Comment:
Not a big deal for this PR, but just a TODO for later. Let's plan on changing all the `is_a()` calls throughout the codebase with `instanceof` which is roughly 50% faster (but all benchmarks are lies!) Lol.

Reply:
The advantage of this is that it is a string and it is probably less risky?